### PR TITLE
[Update] Azure patterns & practices - Freshness pass

### PR DESCRIPTION
**Pull request by ckittel**
- Remove link to github issues and instead use UUF
- Be clear this is "Azure" patterns & practices, not any of the other PnP teams
- Replace martin with our new manager, Luke

### CODE REVIEW

**Comment by ckittel** [comment]
#assign-reviewer:"@skabou"

File: Contribute/content/architecture-center/aac-contribute.md [hunk 1]
```diff
@@ -1,34 +1,26 @@
...
-
-[Azure Architecture Center proposal](https://github.com/MicrosoftDocs/architecture-center/issues/new?title=Content%20suggestion&body=Provide%20a%20detailed%20description%20of%20your%20proposal,%20including%20links%20to%20related%20or%20impacted%20articles,%20and%20any%20relevant%20attachments:)
-
-[Azure Well-Architected Framework proposal](https://github.com/MicrosoftDocs/well-architected/issues/new?title=Content%20suggestion&body=Provide%20a%20detailed%20description%20of%20your%20proposal,%20including%20links%20to%20related%20or%20impacted%20articles,%20and%20any%20relevant%20attachments:)
-
-[Cloud Adoption Framework for Azure proposal](https://github.com/MicrosoftDocs/cloud-adoption-framework/issues/new?title=Content%20suggestion&body=Provide%20a%20detailed%20description%20of%20your%20proposal,%20including%20links%20to%20related%20or%20impacted%20articles,%20and%20any%20relevant%20attachments:)
+If you can't directly propose the change through GitHub, you can also use the [feedback experience](../provide-feedback.md) at the bottom of the article you'd like to see improved. Please leave a detailed description of the change you'd like to see and why.
```
Comment (skabou) [inline Contribute/content/architecture-center/aac-contribute.md:26]:
```suggestion
If you can't directly propose changes through GitHub, you can also use the [feedback experience](../provide-feedback.md) at the bottom of the article you'd like to see improved. Please leave a detailed description of the change you'd like to see and why.
```

**Review by skabou — APPROVED** [review]
Approved - Just suggested a tiny idea for a content change

**Comment by ckittel** [comment]
#sign-off

**Comment by ckittel** [comment]
This is ready for you @jehchow, @carlyrevier, or @Kiranchandratrey

**Review by mikekinsman — APPROVED** [review]

**Comment by ckittel** [comment]
Thanks, Mike

**Pull request merged**